src/cart/crt_internal_types.h

```diff
@@ -89,8 +89,12 @@ struct crt_gdata {
 
 	/* refcount to protect crt_init/crt_finalize */
 	volatile unsigned int	cg_refcount;
-	volatile unsigned int	cg_inited:1,
-				cg_grp_inited:1; /* group initialized */
+
+	/* flags to keep track of states */
+	volatile unsigned int	cg_inited		: 1,
```

Comment:
(style)  Use of volatile is usually wrong: see Documentation/volatile-considered-harmful.txt